src/shared/modules/discovery/discoveryDuck.ts

```diff
@@ -133,21 +145,34 @@ export const discoveryOnStartupEpic = (some$: any, store: any) => {
     .ofType(APP_START)
     .map((action: any) => {
       if (!action.url) return action
+      const { searchParams } = new URL(action.url)
+
       const passedURL =
-        getUrlParamValue('dbms', action.url) ||
-        getUrlParamValue('connectURL', action.url)
+        searchParams.get('dbms') || searchParams.get('connectURL')
+
+      const passedDb = searchParams.get('db')
 
-      const passedDb = getUrlParamValue('db', action.url)
+      if (passedURL) {
+        action.forceURL = decodeURIComponent(passedURL)
+        action.requestedUseDb = passedDb
+      }
+
+      const discoveryURL =
```

Comment:
can we more generically make this just case insensitive?

Reply:
Opting not to since search params are meant to be case sensitive

Reply:
And only do exactly `discoveryURL`